src/main/java/com/animalfinder/service/BreedServiceStub.java

```diff
@@ -10,7 +10,7 @@
 
 @Component
 public class BreedServiceStub implements IBreedService {
-
+	//I like using ID for each breed of animal

```

Comment:
Don't put discussion/rationale in source code.  That's clutter, which is also technical debt.  Discussion belongs in the pull request, where I am writing this statement.